### PR TITLE
piper: update to 0.7.

### PR DESCRIPTION
**Pull request by cultab**
#### Testing the changes
- I tested the changes in this PR: **briefly**, with a compatible gaming mouse

#### Local build testing
- I built this PR locally for my native architecture, (x86-64-glibc)

### CODE REVIEW

**Comment by classabbyamp** [comment]
#39559

**Comment by classabbyamp** [comment]
can you also update libratbag with this? I'll merge it if so

**Comment by cultab** [comment]
Tested again to make sure, still works.

**Comment by classabbyamp** [comment]
adding `dbus` to `checkdepends` and `make_check_pre="dbus-run-session"` fixed the build fail for me

**Comment by cultab** [comment]
Interesting, looking into why it failed here and not locally :/

**Comment by cultab** [comment]
added those changes

**Comment by cultab** [comment]
ACTUALLY added them now :|

**Comment by classabbyamp** [comment]
ok, let's add `make_check=ci-skip # needs system dbus`

**Comment by cultab** [comment]
added `AppSteam` to piper's `depends`

**Comment by classabbyamp** [comment]
AppStream should be a checkdepend i think

File: srcpkgs/libratbag/template [hunk 1]
```diff
@@ -9,13 +9,15 @@ hostmakedepends="pkg-config git swig"
...
-checkdepends="python3-gobject python3-evdev"
+checkdepends="dbus python3-gobject python3-evdev"
 short_desc="DBus daemon to configure gaming mice"
 maintainer="bra1nwave <bra1nwave@protonmail.com>"
 license="MIT"
 homepage="https://github.com/libratbag/libratbag"
 distfiles="https://github.com/libratbag/libratbag/archive/v${version}.tar.gz"
-checksum=9e6ad8d54daef8cc0f44784bdde9bf5a3879d8a66d11e0b990b87266f81f7329
+checksum=c70b6a259693216e6a57ec3087496e8d6df01d4425b7a06592872b7b123e5fcc
+make_check="ci-skip" # needs system dbus
```
Comment (classabbyamp) [inline srcpkgs/libratbag/template:19]:
```suggestion
make_check=extended # needs system dbus
```
ok, this should be the final issue.

Thanks for the prompt changes :)

Comment (classabbyamp) [reply srcpkgs/libratbag/template]: from Manual.md:

> - `make_check` Sets the cases in which the `check` phase is run.
> This option has to be accompanied by a comment explaining why the tests fail.
> Allowed values:
>   - `yes` (the default) to run if `XBPS_CHECK_PKGS` is set.
>   - `extended` to run if `XBPS_CHECK_PKGS` is `full`.
>   - `ci-skip` to run locally if `XBPS_CHECK_PKGS` is set, but not as part of pull request checks.
>   - `no` to never run.

you can set `XBPS_CHECK_PKGS` in `etc/conf`. Extended is for when checks requires some specific system setup, in this case, the system dbus service being enabled.

**Pull request merged**